etc/Dockerfile

```diff
@@ -36,9 +36,18 @@ ENV LANG en_US.UTF-8
 COPY --from=builder /usr/src/grin/target/release/grin /usr/local/bin/grin
 
 WORKDIR /root/.grin
-RUN grin server config && \
+
+ARG grinnet=""
+
+RUN grin ${grinnet} server config && \
     sed -i -e 's/run_tui = true/run_tui = false/' grin-server.toml
 
+VOLUME ["/root/.grin"]
+
 EXPOSE 13413 13414 13415 13416
 
-ENTRYPOINT ["grin", "server", "run"]
+EXPOSE 3413 3414 3415 3416
```

Comment:
Add a comment that these are the mainnet ports.

Reply:
👍